### ISSUE TITLE
:bug: Table Component is not responsive in mobile view

### ISSUE THREAD

**Issue by Adyasha8105** [chakra-ui/chakra-ui#5264]
### Description

Table box is not responsive in the main page

### Link to Reproduction

https://chakra-ui.com/docs/data-display/table

### Steps to reproduce

1. Go to the [link](https://chakra-ui.com/docs/data-display/table):
ss: 
<img width="569" alt="Screenshot 2021-12-19 at 10 45 01 PM" src="https://user-images.githubusercontent.com/54095539/146684191-f269bce4-de71-4574-8078-b9693b1c4ad4.png">

### Chakra UI Version

1.14.1

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by primos63** [comment]
The `Table` is responsive and does fit on a mobile as can be seen [here](https://codesandbox.io/s/boring-cartwright-qw27h?file=/src/App.js).
The issue is with the docs container holding the table.

**Comment by Adyasha8105** [comment]
Yes, The table is responsive. I am talking about the whole component i.e the container only.

**Comment by Adyasha8105** [comment]
Also, what if you add one more column to the table?

<img width="513" alt="Screenshot 2021-12-19 at 11 20 03 PM" src="https://user-images.githubusercontent.com/54095539/146685277-1a4d97b4-6d1f-4a6d-a02b-a535f3400cee.png">
t

**Comment by NadaaFarook** [comment]
We can add the display to scroll.

**Comment by primos63** [comment]
> Also, what if you add one more column to the table?
> 
> <img alt="Screenshot 2021-12-19 at 11 20 03 PM" width="513" src="https://user-images.githubusercontent.com/54095539/146685277-1a4d97b4-6d1f-4a6d-a02b-a535f3400cee.png">

Then you should use `overflow='auto'`.  It's not going to rearrange itself if that is what you are looking for.

As for the container, it's part of the [docs project](https://github.com/chakra-ui/chakra-ui-docs) which is where this issue needs to go.
